contrib/sarplus/python/tests/test_pyspark_sar.py

```diff
@@ -381,7 +381,13 @@ def test_userpred(
     df = spark.createDataFrame(demo_usage_data)
     model.fit(df)
 
-    url = sar_settings["FILE_DIR"] + "userpred_" + file + str(threshold) + "_userid_only.csv"
+    url = (
```

Comment:
This caused an error that has been fixed with https://github.com/recommenders-team/recommenders/pull/1982